### PR TITLE
update all references to editor or editor view

### PR DESCRIPTION
**Pull request by interurban**
Fixes: https://github.com/pulumi/docs/issues/11899

"Editor view" is now called "Document view"
Table view is still the same but ensure we call it "Table view" everywhere and not "Key-value Editor" or "Key-value Table editor"
Both Table view and Document view are "Editors", so we need to update the language accordingly.

### CODE REVIEW

**Review by toriancrane — COMMENTED** [review]
Hi @interurban ,

I just realized the esc env run blog still has the obsolete thumbprint content there:

```
Then click the **Get thumbprint** button. The thumbprint value that you will be presented with is what you can use to verify the certificate of the provider i.e. Pulumi.

{{< notes type="info" >}}
The AWS console generates the thumbprint value on your behalf. However, if you are creating the OIDC provider programmatically, you will need to generate this value yourself and provide the thumbprint value as a part of your resource definition. You can learn more about what a thumbprint is and how to generate/verify it by referring to the [relevant AWS documentation](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_providers_create_oidc_verify-thumbprint.html).
{{< /notes >}}
```

Are you able to include the removal of this part with this PR?

Also I see "Table view" is capitalized but not "document view". Is this intentional?

**Comment by interurban** [comment]
Thanks for the catch and review, yes no prob I'll edit that part of your blog as well.

File: content/blog/esc-key-value-table-editor-launch/index.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
-title: "Introducing the Key-Value Table Editor for Pulumi ESC"
+title: "Introducing the Table view for Pulumi ESC"
```
Comment (arunkumar611) [inline content/blog/esc-key-value-table-editor-launch/index.md:2]:
```suggestion
title: "Introducing the Table Editor for Pulumi ESC"
```

File: content/blog/esc-key-value-table-editor-launch/index.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
 date: 2024-03-26T00:00:00-07:00
 draft: false
-meta_desc: "The Key-Value Table Editor introduces a new visual interface for managing configuration and secrets using Pulumi ESC."
+meta_desc: "The Table view introduces a new visual interface for managing configuration and secrets using Pulumi ESC."
 meta_image: "meta.png"
```
Comment (arunkumar611) [inline content/blog/esc-key-value-table-editor-launch/index.md:6]: meta image need to be updated as well.

File: content/blog/esc-env-run-aws/index.md [hunk 3]
```diff
@@ -122,13 +122,13 @@ After creating your role, make sure to make a note of your role’s ARN value as
...
 
 {{< video title="Creating a new Pulumi ESC environment" src="https://www.pulumi.com/uploads/create-new-environment.mp4" autoplay="true" loop="true" >}}
 
 ### Step 4: Add the AWS provider integration
 
-Once you’ve created your new environment, you will be presented with a split-pane editor view. You’ll want to clear out the default placeholder content in the editor on the left-hand side and replace it with the following code, making sure to replace `<your-oidc-iam-role-arn>` with the value of your IAM role ARN from the previous step:
+Once you’ve created your new environment, you will be presented with a split-pane document view. You’ll want to clear out the default placeholder content in the editor on the left-hand side and replace it with the following code, making sure to replace `<your-oidc-iam-role-arn>` with the value of your IAM role ARN from the previous step:
```
Comment (arunkumar611) [inline content/blog/esc-env-run-aws/index.md:131]: If we are update this blog, we should update the GIFs too. Can torian help with that too?

File: content/blog/esc-key-value-table-editor-launch/index.md [hunk 4]
```diff
@@ -15,34 +15,34 @@ tags:
...
+The Table view we are launching today introduces an alternate visual interface for handling configuration tasks, enabling the direct input of plaintext and secret values.  Key capabilities of the Table Editor include:
 
 - **CRUD Operations**: Perform Create, Read, Update, and Delete (CRUD) operations on your secrets and  plaintext values
 - **Decrypting Secrets**: Safely decrypt secrets one at a time, directly within the editor
 - **Environment Imports**: [Import environments](/docs/esc/environments/#importing-other-environments) and define the order of imports to suit your project's needs
 - **Variables Projections**: Make variables available as [Pulumi Config](/docs/esc/environments/#using-environments-with-pulumi-iac), [environment variables](/docs/esc/environments/#projecting-environment-variables), and files, streamlining your deployment and runtime configuration.
 - **Interpolations and References**: Add references to existing path-values, or add [interpolations](/docs/esc/environments/#interpolating-values) to dynamically generate values
 
-![Key-Value Table Editor](esc-key-value-table-editor.png)
+![Table view](esc-key-value-table-editor.png)
```
Comment (arunkumar611) [inline content/blog/esc-key-value-table-editor-launch/index.md:32]:
```suggestion
![Table Editor](esc-key-value-table-editor.png)
```

**Comment by arunkumar611** [comment]
@interurban  - We need to be mindful where we call it the "Table View" vs. "Table Editor"

When we are directing users and when we talk about the 'feature' itself, we should call it Table Editor. 

When the concept of Editor is already introduced and we are talking about "how you edit" them, then we should mention it as "Table View" and "Document View". 

Let me know if that differentiation makes sense.

File: content/blog/esc-env-run-aws/index.md (hunk 3 above)
Comment (interurban) [reply content/blog/esc-env-run-aws/index.md:131]: Typically we don't go back to edit videos in blogs, but if we create a new one for the docs pages we can retro fix.

File: content/blog/esc-key-value-table-editor-launch/index.md (hunk 2 above)
Comment (interurban) [reply content/blog/esc-key-value-table-editor-launch/index.md:6]: updated.

**Review by thoward — APPROVED** [review]
LGTM. Thanks @interurban !

**Pull request merged**